packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -61,7 +63,7 @@ export interface SuiteState {
 }
 
 const initialState: SuiteState = {
-    online: true,
+    online: navigator.onLine,
```

Comment:
It's not a strong opinion, but I would prefer the previous solution where there is no reference to `navigator` in reducer. Referencing `navigator` in reducer make it harder to reuse in different environments. It's not a big deal for, React Native have it polyfilled, but it wouldn't work, for example, for server side rendering (that we don't currently use).

Reply:
Is there any user benefit from this change? From the code perspective, I think the original way of setting initial online state was better.

Reply:
I just wanted to remove the "unnecessary" redux action, forgot about native.